auth-table-based-service/src/main/java/io/stargate/auth/table/AuthTableBasedService.java

```diff
@@ -258,4 +261,34 @@ public StoredCredentials validateToken(String token) throws UnauthorizedExceptio
 
     return storedCredentials;
   }
+
+  @Override
+  public ResultSet executeDataReadWithAuthorization(
```

Comment:
WDYT about `authorizedRead(...)` for brevity?

Reply:
Sure, I can change to `authorizedDataRead` to distinguish it from `authorizedSchemaRead`